lambda.tf

```diff
@@ -49,22 +44,25 @@ resource "aws_iam_role" "lambda_role" {
   assume_role_policy = file("${path.module}/policies/LambdaBasicExecution.json")
 }
 
-# resource "aws_iam_role_policy" "lambda_iam_role_policy" {
-#   name   = "lambda_iam_role_policy"
-#   role   = aws_iam_role.lambda_role.name
-#   policy = file("${path.module}/policies/Watchlog.json")
-# }
-
-resource "aws_iam_role_policy" "lambda_iam_role_policy" {
-  name   = "lambda_iam_role_policy"
-  role   = aws_iam_role.lambda_role.name
-  policy = data.template_file.example.rendered
+data "aws_iam_policy_document" "policy_lambda_logs" {
+  statement {
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents"
+    ]
+    effect    = "Allow"
+    resources = ["arn:aws:logs:*:*:*"]
```

Comment:
` resources = [aws_cloudwatch_log_group.lambda_cwgroup.arn]`
test this to ensure the lambda is still able to write logs

Reply:
Done, commit: 8f78c78f9c18aca02ee307e6c070f7c8905196e4